website/src/pages/messages/index.tsx

```diff
@@ -24,7 +27,9 @@ const MessagesDashboard = () => {
       <SimpleGrid columns={[1, 1, 1, 1, 1, 2]} gap={4}>
         <Box>
           <Text className="text-2xl font-bold" pb="4">
-            {t("recent_messages")}
+            {t("recent_messages", {
+              language: new Intl.DisplayNames([currentLanguage], { type: "language" }).of(currentLanguage),
```

Comment:
Maybe it would be a good idea to extract this to a helper method since we use it now in more than one place?

Reply:
Yes, that would make sense.  I'll try to do that in a follow up PR, not this one.